site/content/docs/user/known-issues.md

```diff
@@ -293,7 +293,7 @@ You can work around this by changing the `FirewallBackend` in the `/etc/firewall
 
 ```console
 sed -i /etc/firewalld/firewalld.conf 's/FirewallBackend=.*/FirewallBackend=iptables/'
-systemctl restart docker
+systemctl restart firewalld
```

Comment:
Line above says `restarting docker` , seems the user submitted it found that restarting docker solved the problem.
Is ok restarting only firewalld?

Reply:
please change line 292 too :)

Reply:
> Line above says `restarting docker` , seems the user submitted it found that restarting docker solved the problem.
> Is ok restarting only firewalld?

yep, I just needed to restart firewalld